source/ftrack_connect_pipeline_qt/ui/assembler/assembler.py

```diff
@@ -561,9 +561,8 @@ def __init__(self, assembler_widget, parent=None):
         :param assembler_widget: :class:`~ftrack_connect_pipeline_qt.ui.assembler.base.AssemblerBaseWidget` instance
         :param parent: The parent dialog or frame
         '''
-        self._asset_widget_class = DependencyComponentWidget
         super(DependenciesListWidget, self).__init__(
-            assembler_widget, parent=parent
+            assembler_widget, DependencyComponentWidget, parent=parent
```

Comment:
This is a bit confusing. Why are you passing "DependencyComponentWidget" into the DependenciesListWidget init instead of importing it directly on the DependenciesListWidget?

Reply:
Because I have consolidated the AssetList so it stores the class and enforcing using when instantiating the item widgets, look at line 633 how all AssetListWidget descendants uses the self.item_widget_class